lib/aad_oauth.dart

```diff
@@ -57,6 +57,11 @@ class AadOAuth {
     AadOAuth(_config);
   }
 
+  void init(Rect screenSize) async {
```

Comment:
oh sorry, didn't think about it, but probably this should be optional, because some people would initialize it in the main function, where the screenSize is not yet known sometimes

so optional screenSize and if not passed, don't call `setWebViewScreenSize`

Reply:
It is actually the case in our app. I completely removed screen initialization if it's ok for you